tools/toolchain/scripts/stage4/install_cosma.sh

```diff
@@ -35,16 +40,17 @@ case "$with_cosma" in
         echo "COSMA-v${cosma_ver}.tar.gz is found"
       else
         download_pkg_from_cp2k_org "${cosma_sha256}" "COSMA-v${cosma_ver}.tar.gz"
+        download_pkg_from_cp2k_org "${costa_sha256}" "COSTA-v${costa_ver}.tar.gz"
+        download_pkg_from_cp2k_org "${tiled_mm_sha256}" "Tiled_MM-v${tiled_mm_ver}.tar.gz"
```

Comment:
I think this should be `Tiled-MM`, ie. with `-` instead of `_`.

Reply:
thanks.